js/src/views/Web/web.css

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2015, 2016 Ethcore (UK) Ltd.
+/* This file is part of Parity.
+/*
+/* Parity is free software: you can redistribute it and/or modify
+/* it under the terms of the GNU General Public License as published by
+/* the Free Software Foundation, either version 3 of the License, or
+/* (at your option) any later version.
+/*
+/* Parity is distributed in the hope that it will be useful,
+/* but WITHOUT ANY WARRANTY; without even the implied warranty of
+/* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+/* GNU General Public License for more details.
+/*
+/* You should have received a copy of the GNU General Public License
+/* along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+.wrapper {
+  border: 0;
+  position: absolute;
+  height: 100%;
+  width: 100%;
+}
+.loading {
```

Comment:
Would prefer a space between the blocks, doesn't cost anything but aids in readability. (Same for at the end of this block). -

```
.wrapper {
  ...
}

.loading {
  ...
}

.url {
  ...
}
```